backend/baseclass.pm

```diff
@@ -16,7 +16,7 @@ use File::Copy 'cp';
 use File::Basename;
 use Time::HiRes qw(gettimeofday time tv_interval);
 use Try::Tiny;
-use POSIX qw(_exit :sys_wait_h);
+use POSIX qw(_exit waitpid WNOHANG);
```

Comment:
Note that this is not required on TW. However, with the Perl version on Leap 15.3/15.4 the test otherwise fails. I've seen this in OBS but now also in a local container.

The following would work as well:
```suggestion
use POSIX qw(_exit :sys_wait_h waitpid);
```

So it is `waitpid` that is the undefined subroutine in `Undefined subroutine called at /tmp/os-autoinst/backend/baseclass.pm line 1306.`. However, for the sake of consistency it is likely best to just include both symbols explicitly and avoid use of the cryptic `:sys_wait_h` altogether.